server/src/instant/dash/routes.clj

```diff
@@ -1177,6 +1177,12 @@
     (instant-user-refresh-token-model/delete-by-id! {:id token})
     (response/ok {})))
 
+(defn active-sessions-get [_]
+  (let [total-count (->> (machine-summaries/get-all-num-sessions (eph/get-hz))
```

Comment:
Can you cache this on the server? I'm worried that we'll get a ton of these requests and we'll overwhelm hazelcast.

Reply:
cc @dwwoelfel added!